src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projecthome/ProjectHomeView.kt

```diff
@@ -0,0 +1,65 @@
+package org.wycliffeassociates.otter.jvm.app.ui.projecthome
+
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.geometry.Pos
+import org.wycliffeassociates.otter.jvm.app.UIColorsObject.Colors
+import org.wycliffeassociates.otter.jvm.app.widgets.projectcard
+import tornadofx.*
+
+class ProjectHomeView : View() {
+
+    val viewModel: ProjectHomeViewModel by inject()
+    override val root = borderpane {
+        top = hbox { alignment = Pos.CENTER_RIGHT
+            button("Refresh") {
+                prefWidth = 232.0
+                prefHeight = 40.0
+                action {
+                    viewModel.getAllProjects()
```

Comment:
Would it be possible to move this to the `onDock` method to automatically refresh whenever the view appears?

Reply:
yes. it will go in the onDock. reason i have that button there for right now is i was having problems getting the projects to refresh on the homepage once the wizard closes. the wizard was closing and docking the new view before the database had been updated with the new project, so i couldn't the getAllProjects wasn't getting all the projects at the time. I figured i'd sandbox myself on that, get other stuff done and then come back to it once everything else was filled out